travis-ci/build_steps.sh

```diff
@@ -98,11 +101,16 @@ function do_build_lib {
             local bitness=64
             local target_flags="TARGET=ARMV8"
             ;;
+        arm64)
+            local bitness=64
+            local target_flags="TARGET=VORTEX"
+            ;;
         s390x)
             local bitness=64
             ;;
         ppc64le)
             local bitness=64
+            local target_flags="TARGET=POWER8"
```

Comment:
I wonder if this change will impact the future numpy / scipy wheels built for ppc64le. Those wheels are typically built on a PPC host (without cross-compilation) so maybe this won't change a thing (but I agree it's best to make the target architecture explicit).

Reply:
> Those wheels are typically built on a PPC host (without cross-compilation) so maybe this won't change a thing

It will if the host becomes a power9 or power10 some day.